plugins/memcached/CMakeLists.txt

```diff
@@ -31,20 +31,35 @@ if (MEMCACHED)
         # until then, we build the required libraries from source
 
         set(LIBMEMCACHED_VERSION "1.0.10")
+        set(LIBMEMCACHED_URL https://launchpad.net/libmemcached/1.0/${LIBMEMCACHED_VERSION}/+download/libmemcached-${LIBMEMCACHED_VERSION}.tar.gz)
+        file(DOWNLOAD
```

Comment:
Shouldn't we check that a local version (>= min version) doesn't exist first before attempting to download and, at that, forcing the min version to be used? I think the downloaded version should be configurable as a cmake option.